site-src/api-types/httproute.md

```diff
@@ -165,6 +165,31 @@ Service:
 Reference the [backendRef][backendRef] API documentation for additional details
 on `weight` and other fields.
 
+#### Timeouts (optional)
+
+??? example "Experimental Channel in v1.0.0+"
+
+    HTTPRoute timeouts are part of the Experimental Channel in `v1.0.0+`.
+
+HTTPRoute Rules include a `Timeouts` field. If unspecified, timeout behavior is implementation-specific.
+
+There are 2 kinds of timeouts that can be configured in an HTTPRoute Rule:
+
+1. `request` is the timeout for the Gateway API implementation to send a response to a client HTTP request. Whether the Gateway starts the timeout before or after the entire client request stream has been received, is implementation dependent. This timeout is intended to cover as close to the whole request-response transaction as possible, although an implementation MAY choose to start the timeout after the entire request stream has been received instead of immediately after the transaction is initiated by the client.
```

Comment:
This is saying the implementation dependent part twice.

```suggestion
1. `request` is the timeout for the Gateway API implementation to send a response to a client HTTP request. This timeout is intended to cover as close to the whole request-response transaction as possible, although an implementation may choose to start the timeout after the entire request stream has been received instead of immediately after the transaction is initiated by the client.
```